### DISCUSSION TITLE
Release of mtexTools library

### DISCUSSION THREAD

**Discussion by frankNiessen** [ORTools4MTEX/ORTools#36] [Announcements]
Dear ORTools users, 

I want to announce the release of a [new collated library of MTEX functions & scripts](https://github.com/AzdiarGazder/mtexTools) by my collaborator @AzdiarGazder. 
He has drawn together a number of utility and applied technical functions that are each documented with a short demo script. 
Some of these might come in handy in your MTEX / ORTools workflow, so make sure to check them out. 

Best wishes
Frank

**Comment by NovakEvana** [comment]
Awesome, gonna try them out soon. Can't get enough time to get back to you yet. Thanks to all of you for this!

**Comment by AzdiarGazder** [comment]
Hi @frankNiessen,

Thank you for the plug. :)
I hope these additional Mtex tools/scripts are useful to everyone.

I will continue to add more scripts that improve on Mtex's general functionality to [**mtexTools**](https://github.com/AzdiarGazder/mtexTools) into the future.
So please add [**mtexTools**](https://github.com/AzdiarGazder/mtexTools) to your list of Github Dektop repositories to keep it updated. 

Warm regards,
Azdi
